runtime-tests/index.ts

```diff
@@ -0,0 +1,35 @@
+import { ArgumentParser } from 'argparse';
+
+const config = {
+	chain: 'polkadot',
```

Comment:
Why do we start with polkadot?

Reply:
'polkadot' currently just sits as a placeholder for the key value pair.